README.md

```diff
@@ -1,4 +1,63 @@
-# CSharp-From-Zero-To-Hero
-[![Build Status](https://dev.azure.com/almantusk/From-Zero-To-Hero/_apis/build/status/Almantask.CSharp-From-Zero-To-Hero?branchName=master)](https://dev.azure.com/almantusk/From-Zero-To-Hero/_build/latest?definitionId=5&branchName=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a358adf21c2442cd91a4827a50213ff1)](https://www.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Almantask/CSharp-From-Zero-To-Hero&amp;utm_campaign=Badge_Grade)  
-C# boot camp
+# C#: From Zero To Hero
```

Comment:
As you can see, you're trying to merge to csinn/master. However, you should be merging with csinn/Chapter1/Homework/1And2, otherwise it will be harder to tell what changes you made compared to the original homework branch.